CHANGELOG

```diff
@@ -1,3 +1,7 @@
+2015-09-23: v0.3.0
+  Support Mesos 0.25 NetworkInfo as an IPSource (#266, #267, #268)
```

Comment:
People will read "0.25" is supported. Write something like "Experimental support ....".